cmd/init.go

```diff
@@ -1,18 +1,20 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	"github.com/celestiaorg/celestia-node/node"
 )
 
 // Init constructs a CLI command to initialize Celestia Node of the given type 'tp'.
 // It is meant to be used a subcommand and also receive persistent flag name for repository path.
-func Init(repoName string, tp node.Type) *cobra.Command {
+func Init(repoFlagName string, tp node.Type) *cobra.Command {
 	if !tp.IsValid() {
 		panic("cmd: Init: invalid Node Type")
 	}
-	if len(repoName) == 0 {
+	if len(repoFlagName) == 0 {
```

Comment:
won't `len(repoFlagName)` never == 0 ? We're removing dev mode anyway which is the only thing that passes `""` to this function.

Reply:
Just a sanity check, as `Start` is exported and might be used outside. Does not hurt. Will revisit this when will delete Dev mode. Resolved?

Reply:
@renaynay

Reply:
BTW, is there an issue to track removal of dev mode? 
Can this be closed then: https://github.com/celestiaorg/celestia-node/pull/176#issuecomment-985824614

Reply:
Implicitly tracked here: https://github.com/celestiaorg/celestia-node/issues/250

Resolving.